news/2022-10-01-osumonthly-3.md

```diff
@@ -0,0 +1,164 @@
+
+<iframe width="100%" height="315" src="https://www.youtube.com/watch?v=SonwRhyol28" frameborder="0" allowfullscreen></iframe>
+
+However, this certainly hasn't been the only impressive play this month, as we've seen [ttobas](https://osu.ppy.sh/users/2084568) set a [new NM pp record](https://osu.ppy.sh/scores/fruits/209125694) on the recently ranked [Storms of Ages](https://osu.ppy.sh/beatmapsets/1673971#fruits/3419551), a map with the highest star rating in CTB.
+
+## Did you know?
+
+[LilyPichu](https://osu.ppy.sh/users/17679), a now prominent content creator loaned her voice to several skin creators back in the early 2010's. For example, you can check out the [Nexus Slate & Nexus Ivory](https://osu.ppy.sh/community/forums/topics/55948?n=1) pair of skins which include her voice in places such as the map start countdown and applause at the end of a map.
+
+See if you can find more skins that include her voice and let us know in the comments!
+
+## Monthly community quiz
+
+We decided to make a small change to how the quiz works as we prepare to start writing harder questions. Starting from this month, there will be a time limit of 1 month (since the release of the news post) to answer the question, if the question is not answered by the time next month's osu!monthly is released, the reward will not be claimable anymore.
+We will also be posting the answers of last month's quiz as we feel it only makes sense regardless of whether a correct answer was provided or not.
+
+Now that that's out of the way, the question we have prepared for you is:
+
+TBA
+
+## Things to look out for in October
+
+- The [osu! university](https://discord.com/invite/QubdHdnBVg) rises from its ashes with the [osu! improvement month](https://osu.ppy.sh/community/forums/topics/1649135?n=1) event that stretches over the whole month of October. Interviews, tournaments, coaching, and giveaways are planned, and it would definitely be a mistake not to take advantage of this opportunity to socialize with the community, or to follow the events of your favorite game!
+- After several weeks of hard work by creators and the voting phase coming and going, we should soon be able to see the results of the [triangles Beatmapping Contest](/wiki/Contests/triangles_Beatmapping_Contest). Who was able to combine the best storyboard, hitsound, and mapping around the triangles theme? We can't wait to see!
+- The voting stage for the [Chromatic Alteration skinning contest](https://osu.ppy.sh/home/news/2022-07-18-skinning-contest-chromatic-alteration-announcement#challenge:-chromatic-alteration) has concluded! With 23 entries and only 3 votes per player, hard decisions had to be made when it came to figuring out which entries deserve votes the most, and we are all eagerly awaiting to see the final results!
```

Comment:
dont see any reason why its linking to the challenge section, should probably just link to the newspost without the #id